app/mailers/gov_notify_mailer.rb

```diff
@@ -1,4 +1,12 @@
 class GovNotifyMailer < GovukNotifyRails::Mailer
+  def send_expression_of_interest_confirmation_email(application)
+    set_template(ENV["EXPRESSION_OF_INTEREST_CONFIRMATION_TEMPLATE_ID"])
```

Comment:
reminder for us to create the templates (staging and prod) and add the flags in the environments as well